src/components/CGroupHeader.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <tr @click="onToggle" class="text-xs hover:bg-blue-100">
+    <td class="border px-4 py-2">
+      <div class="flex">
+        <img v-if="isOpened" src="@/assets/img/expand_more.svg" width="20rem" />
+        <img v-else src="@/assets/img/expand_less.svg" width="20rem" />
+        <pre class="text-xs ml-2 whitespace-normal">{{ groupId }}</pre>
+      </div>
+    </td>
+    <td class="border px-4 py-2">
+      {{ remainingTime }}
+    </td>
+    <td class="border px-4 py-2">
+      {{ progress | percentage }}
+    </td>
+    <td class="border px-4 py-2">
+      {{ messageCount }}
+    </td>
+    <td class="border px-4 py-2">
+      <ul>
+        <li :key="index" v-for="(actor, index) in uniqueActors">
```

Comment:
i would prefer to separate it with, because if there is a lot of actors, the line would becomve very high.